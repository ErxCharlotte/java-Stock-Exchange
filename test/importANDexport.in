ADD trader1 100.00
ADD trader2 100.00
IMPORT trader3 ABC 100
IMPORT trader1 ABC -1
import trader1 ABC 100
EXPORT trader3 ABC 100
EXPORT trader1 ABC -1
EXPORT trader1 LOL 100
EXPORT trader1 ABC 200
EXPORT trader1 ABC 50
EXPORT trader1 ABC 50
EXIT