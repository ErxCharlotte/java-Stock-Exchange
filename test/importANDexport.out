$ Success.
$ Success.
$ No such trader in the market.
$ Could not import product into market.
$ Trader now has 100.00 units of ABC.
$ No such trader in the market.
$ Could not export product out of market.
$ Could not export product out of market.
$ Could not export product out of market.
$ Trader now has 50.00 units of ABC.
$ Trader now has no units of ABC.
$ Have a nice day.